cloudwatch.tf

```diff
@@ -30,3 +34,22 @@ resource "aws_cloudwatch_event_target" "ecs_scheduled_task" {
     }
   }
 }
+
+# Cloudwatch Log Group
+# ------------------
+resource "aws_cloudwatch_log_group" "ecs_scheduled_task" {
+  name              = local.cloudwatch_log_group_name
+  retention_in_days = var.cloudwatch_log_group_retention
+  kms_key_id        = module.cloudwatch_log_group_kms.key_arn
```

Comment:
Have you tested the use-case when kms is not enabled ?

Reply:
this value is controlled by [kms module](https://github.com/cloudposse/terraform-aws-kms-key/blob/0de0b81b77b20b9397b27c26cb51bcdbae1d3098/outputs.tf#L6-L9) from cloudposse

but reference to the policy for this module had to be fixed with `one(data.aws_iam_policy_document.cloudwatch_log_group_kms[*].json)`